storage/innobase/log/log0chkp.cc

```diff
@@ -1189,6 +1189,8 @@ void log_calc_max_ages(log_t &log) {
 
   const lsn_t limit = log.lsn_capacity_for_free_check;
 
+  log.max_checkpoint_age.store(log_get_free_check_capacity(log));
```

Comment:
[optional] There is actually no need to use store()/load() on std::atomic if you are using default parameter for `memory_order` (sequentially consistent). operator = / operator T overloads work exactly the same way
```suggestion
  log.max_checkpoint_age = log_get_free_check_capacity(log);
```